json-guides/circuit-breaker.json

```diff
@@ -303,10 +303,10 @@
               "Now that you learned about circuit breakers and fallbacks, you can explore the parameters in the <code>@CircuitBreaker</code> annotation and see the circuit in action.",
               "<br>You learned about the following parameters:",
               "<ul>",
-              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made before determining if the circuit should trip. This also specifies the size of the rolling window. The default is <code>20</code> requests.",
+              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made to the service before determining whether the circuit trips. This parameter also specifies the size of the rolling window. The default is <code>20</code> requests.",
```

Comment:
`This parameter` is used here, while `The parameter` is used in the step we first introduce requestVolumeThreshold.

Reply:
Changed.